csv_formatter.py

```diff
@@ -0,0 +1,309 @@
+    data_cell = str(data_cell).lower()
+    return True if data_cell == "yes" else False
+
+
+with open("homework.csv", encoding="UTF-8") as input_file:
+    data = pd.read_csv(input_file, sep=",", dtype="object")
+
+    # Rename columns which will appear in formatted.csv
+    data.rename(
+        columns={
+            "item number": "manufacturer_sku",
+            "item weight (pounds)": "weight",
+            "item depth (inches)": "length",
+            "item width (inches)": "width",
+            "item height (inches)": "height",
+            "wholesale ($)": "cost_price",
+            "map ($)": "min_price",
+            "item category": "product__product_class__name",
+            "brand": "product__brand__name",
+            "description": "product__title",
+            "long description": "product__description",
+            "selling point 1": "product__bullets__0",
+            "selling point 2": "product__bullets__1",
+            "selling point 3": "product__bullets__2",
+            "selling point 4": "product__bullets__3",
+            "selling point 5": "product__bullets__4",
+            "selling point 6": "product__bullets__5",
+            "selling point 7": "product__bullets__6",
+            "carton count": "product__multipack_quantity",
+            "country of origin": "product__country_of_origin__alpha_3",
+            "furniture arm height (inches)": "attrib__arm_height",
+            "bulb 1 included": "attrib__bulb_included",
+            "bulb 1 type": "attrib__bulb_type",
+            "primary color family": "attrib__color",
+            "cord length (inches)": "attrib__cord_length",
+            "licensed by": "attrib__designer",
+            "item finish": "attrib__finish",
+            "conversion kit option": "attrib__kit",
+            "item materials": "attrib__material",
+            "bulb 1 count": "attrib__number_bulbs",
+            "outdoor": "attrib__outdoor_safe",
+            "furniture seat height (inches)": "attrib__seat_height",
+            "shade/glass description": "attrib__shade",
+            "switch type": "attrib__switch_type",
+            "bulb 1 wattage": "attrib__wattage",
+            "furniture weight capacity (pounds)": "attrib__weight_capacity",
+            "carton 1 weight (pounds)": "boxes__0__weight",
+            "carton 1 length (inches)": "boxes__0__length",
+            "carton 1 height (inches)": "boxes__0__height",
+            "carton 1 width (inches)": "boxes__0__width",
+            "carton 2 weight (pounds)": "boxes__1__weight",
+            "carton 2 length (inches)": "boxes__1__length",
+            "carton 2 height (inches)": "boxes__1__height",
+            "carton 2 width (inches)": "boxes__1__width",
+            "carton 3 weight (pounds)": "boxes__2__weight",
+            "carton 3 length (inches)": "boxes__2__length",
+            "carton 3 height (inches)": "boxes__2__height",
+            "carton 3 width (inches)": "boxes__2__width",
```

Comment:
What happens if the file which is input has a "carton 4" set of columns?

I would prefer if columns were collected into a list or vector-like.